docs/data/xrootd-authorization.md

```diff
@@ -0,0 +1,141 @@
+In this document, we will cover the security option supported in the OSG:
+
+-   [`xrootd-lcmaps`](#recommended-security-option-xrootd-lcmaps-authorization): Using the LCMAPS callout to use
+    X509-based authentication
+
+Note: On the data nodes, the files will actually be owned by unix user `xrootd` (or other daemon user), not as the user
+authenticated to, under most circumstances. 
+XRootD will verify the permissions and authorization based on the user that the security plugin authenticates you to,
+but, internally, the data node files will be owned by the `xrootd` user.
+
+#### Authorization file
+
+In order to add security to your cluster you will need to add "auth\_file" on the your data server node. 
+Create `/etc/xrootd/auth_file` :
+
+```file
+# This means that all the users have read access to the datasets 
+u * %RED%/data/xrootdfs%ENDCOLOR% lr
+
+# This means that all the users have full access to their private dirs 
+u = %RED%/data/xrootdfs/%ENDCOLOR%@=/ a
+
+# This means that this privileged user can do everything 
+# You need at least one user like that, in order to create the 
+# private dir for each user willing to store his data in the facility 
+u xrootd %RED%/data/xrootdfs%ENDCOLOR% a 
+```
+
+Here we assume that your storage path is `/data/xrootdfs` (same as in the
+previous example).
+
+Change file ownership (if you have created file as root):
+
+```console
+root@host # chown xrootd:xrootd /etc/xrootd/auth_file
+```
+
+
+This file is a flat file of the following form:
+
+``` file
+idtype id path privs
+```
+
+| Item   | Description                                                                                                                           |
+|--------|---------------------------------------------------------------------------------------------------------------------------------------|
+| idtype | Type of id. Use `u` for username, `g` for group, etc.                                                                                 |
+| id     | Username (or groupname). Use `*` for all users or `=` for user-specific capabilities, like home directories                           |
+| path   | The path prefix to be used for matching purposes                                                                                      |
+| privs  | Letter list of privileges: `a - all ; l - lookup ; d - delete ; n - rename ; i - insert ; r - read ; k - lock (not used) ; w - write` |
+
+Examples:
+
+            # any user can read files everywhere _except_ under /private
+            u * -rl /private / rl
+            # users can do anything to files and dirs under their home directories
+            u = /home/@=/ a
+            # VOMS group uscms can do anything to files and dirs under /uscms
+            g uscms /uscms a
+
+
+Note: specific paths need to be specified _before_ generic paths, i.e.
+
+            u * rl / -rl /private
+
+does not work.
```

Comment:
* Let's make this a `!!! note`
* It seems like more of an `e.g.`
* Let's make this a single line instead of a sentence split by a code block

Reply:
Completely agree with the "e.g." remark; the original "i.e." text was just taken from [this comment](https://github.com/opensciencegrid/docs/pull/458#discussion_r229325526).

Updated.